packages/libraries/cli/src/commands/app/create.ts

```diff
@@ -129,21 +145,29 @@ export default class AppCreate extends Command<typeof AppCreate> {
 
     let counter = 0;
 
-    for (const [hash, body] of Object.entries(validationResult.data)) {
+    for (const [hash, body] of Object.entries(operations)) {
       buffer.push({ hash, body });
       await flush();
       counter++;
     }
 
     await flush(true);
 
-    this.log(
-      `\nApp deployment "${flags['name']}@${flags['version']}" (${counter} operations) created.\nActive it with the "hive app:publish" command.`,
-    );
+    const message = `App deployment "${flags['name']}@${flags['version']}" (${counter} operations) created.\nActivate it with the "hive app:publish" command.`;
+    this.log(message);
+    return this.successData({
+      message,
+      effect: 'executed',
+      data: {
+        id: result.createAppDeployment.ok.createdAppDeployment.id,
+        name: flags['name'],
+        version: flags['version'],
+      },
+    });
   }
 }
 
-const ManifestModel = z.record(z.string());
+const ManifestModel = Typebox.Record(Typebox.String(), Typebox.String());
```

Comment:
This was a simple Zod use-case so I replaced with Typebox which we're moving towards now.